ProcessMaker/Http/Controllers/Api/UserController.php

```diff
@@ -331,6 +344,34 @@ public function update(User $user, Request $request)
         return response([], 204);
     }
 
+    /**
+     * Validate the phone number for SMS two-factor authentication.
+     *
+     * @param User $user 
+     * @param mixed $number 
+     */
+    private function validateCellPhoneNumber(User $user, $number)
+    {
+        $methods = $user->getValid2FAPreferences();
+        $hasSMS2FA = in_array(TwoFactorAuthentication::SMS, $methods);
+        $isValid = !$hasSMS2FA || preg_match('/^[+\.0-9x\)\(\-\s\/]+$/', $number);
+        if (!$isValid) {
+            return response([
+                'message' => __(
+                    'A valid Cell phone number is required for SMS two-factor authentication.'
```

Comment:
this label is missing in the language json files

Reply:
Missing translations were added.